src/uct/sm/scopy/base/scopy_ep.c

```diff
@@ -55,11 +63,10 @@ uct_scopy_ep_prepare_tx(uct_ep_h tl_ep, const uct_iov_t *iov,
         return UCS_ERR_NO_MEMORY;
     }
 
-    tx->comp            = comp;
-    tx->rkey            = rkey;
-    tx->remote_addr     = remote_addr;
-    tx->super.op        = tx_op;
-    tx->iov_cnt         = 0;
+    tx->rkey        = rkey;
+    tx->remote_addr = remote_addr;
+    tx->iov_cnt     = 0;
+    uct_scopy_ep_tx_init_common(tx, tx_op, comp);
```

Comment:
minor: i'd call uct_scopy_ep_tx_init_common() first since it's sort of "base initialization"